SampleViewer/SampleViewer.pro

```diff
@@ -37,7 +37,7 @@ exists($$PWD/../../../DevBuildCpp.pri) {
   # use the Esri dev build script
   include ($$PWD/../../../DevBuildCpp.pri)
   # include the toolkitcpp.pri, which contains all the toolkit resources
-  include($$PWD/../../toolkit/uitools/toolkitcpp.pri)
+  include($$PWD/../../toolkit/uitools/toolkitcpp/toolkitcpp.pri)
```

Comment:
This avoids a toolkit warning. This file was recently moved into `uitools/toolkitcpp`. There is still a `uitools/toolkitcpp.pri`, but it logs a warning and says to use the new file directly.

FYI @GuillaumeBelz